tests/test_outbox.py

```diff
@@ -0,0 +1,126 @@
+import datetime
```

Comment:
`datetime` has inconsistent usage throughout the files, they should be `from datetime import datetime`.

Reply:
Good call out, done ✅